src/components/tasks/single/singleTaskContent.tsx

```diff
@@ -65,8 +65,8 @@ const SingleTaskContent = ({ task }: SingleTaskProps) => {
                 {attachments.map((a, i) => {
                     const filename = a.file.split('/').pop();
                     return (
-                        <div className="pr-2">
-                            <Attachment key={i} href={a.file} onTaskModal>
+                        <div key={i} className="pr-2">
```

Comment:
This div can be replaced by putting `space-x-` style in the parent div. Please verify ;)

Reply:
style-x- doesn't work

Reply:
Oh, probably that's why somebody used padding here.